packages/core/stories/text/text.doc.stories.mdx

```diff
@@ -32,22 +32,13 @@ We've created a base Text component (that contains our key typography styles and
 
 You can also create your own text components using the props and CSS variables to configure them.
 
-# API
+# Features
 
-```
-import { Text, H1, H2, H3, H4, Label, Display1, Display2, Display3 } from "@jpmorganchase/uitk-core";
-```
-
-## Props
-
-<ArgsTable of={Text} />
+## Variant
```

Comment:
```suggestion
### Variant
```
But also, in Button we have a title Button variants instead. We should be consistent.